_includes/price-table.html

```diff
@@ -1,4 +1,9 @@
 {% assign category = include.category %}
+{% assign approver = mitoc-approval@mit.edu %}
```

Comment:
We might need quotes here? Maybe that's why the assignment doesn't seem to be working?